test/config_parser_SUITE.erl

```diff
@@ -1585,23 +1563,22 @@ mod_global_distrib(_Config) ->
         {message_ttl, 42},
         {redis, RedisOpts}
        ]), T(Base)),
-    ?eqf(modopts(mod_global_distrib,
```

Comment:
This whole test case needs:

- Splitting per subsection into separate test cases.
- Testing with only required options as Base.

I know it works as it is, but we are reworking them all now... see the PR for part 3/4 for reference.